src/store/actionCreators/user.ts

```diff
@@ -0,0 +1,53 @@
+import { ThunkAction } from 'redux-thunk';
+import { getUser } from '../../api';
+import { RootState } from '../store';
+import {
+  FETCH_USER_REQUEST,
+  FETCH_USER_SUCCESS,
+  FETCH_USER_FAILURE,
+} from '../actionTypes/user';
+import { User } from '../../types/User';
+
+type FetchUserRequestAction = {
+  type: 'FETCH_USER_REQUEST';
+};
+
+type FetchUserSuccessAction = {
+  type: 'FETCH_USER_SUCCESS';
+  payload: User;
+};
+
+type FetchUserFailureAction = {
+  type: 'FETCH_USER_FAILURE';
+  payload: string;
+};
+
+/* eslint-disable-next-line */
+type Action = FetchUserRequestAction | FetchUserSuccessAction | FetchUserFailureAction;
+
+export const fetchUserRequest = (): FetchUserRequestAction => ({
+  type: FETCH_USER_REQUEST,
+});
+
+export const fetchUserSuccess = (user: User): FetchUserSuccessAction => ({
+  type: FETCH_USER_SUCCESS,
+  payload: user,
+});
+
+export const fetchUserFailure = (error: string): FetchUserFailureAction => ({
+  type: FETCH_USER_FAILURE,
+  payload: error,
+});
+
+/* eslint-disable-next-line */
```

Comment:
Remove this rule and just separate on different line the code.

Reply:
impossible, i can not do it